examples/multilingual/content/uudis/_index.et.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "Uudised"
+url = "/uudised"
```

Comment:
I think we should remove what I consider superflous front matter in this example. In this particular file that would be "url" and "listing". If you could also remove "url" from all other content files. We should encourage using the URLs that Hugo provides out of the box.

Reply:
I have removed the front matter now. Also organized the content files to get a more coherent structure of the example site.